index.html

```diff
@@ -485,17 +482,17 @@ <h3>Character</h3>
         <h3>Script Event</h3>
         <p>An <dfn>Script Event</dfn> object represents dialogue, on screen text or audio descriptions to be spoken and has the following properties:
           <ul>
-            <li>An <dfn>Script Event Identifier</dfn> which is unique in the script</li>
-            <li><dfn>Begin</dfn> and <dfn>End</dfn> properties providing the <a>Script Event</a> time interval in the programme timeline
+            <li>A mandatory <dfn>Script Event Identifier</dfn> which is unique in the script</li>
+            <li>Mandatory <dfn>Begin</dfn> and <dfn>End</dfn> properties providing the <a>Script Event</a> time interval in the programme timeline
```

Comment:
Not quite clear that both properties are mandatory, somehow.

```suggestion
            <li>A mandatory <dfn>Begin</dfn> property and a mandatory <dfn>End</dfn> property that together define the <a>Script Event</a>'s time interval in the programme timeline
```